.github/workflows/push.yaml

```diff
@@ -0,0 +1,64 @@
+name: push
+
+on:
+  push:
+    branches:
+      - master
+
+env:
+  TEST_IMAGE: 'ghcr.io/${{ github.repository_owner }}/trivy-test-images' # used in Makefile
+  GH_USER: "aqua-bot"
+
+jobs:
+  changed_files:
+    runs-on: ubuntu-latest
+    name: Push changed images
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
```

Comment:
Any reasons? The default value is 1. Isn't good for us?

Reply:
You are right. We can use default value. Changed in https://github.com/aquasecurity/trivy-test-images/pull/30/commits/0e5f060df03d8db0e1b00533eeafa9e8743b5509